index.js

```diff
@@ -23,11 +23,10 @@ server.get("/", (request, response) => {
   const listagem = []
   for (slug in artigos) {
     const artigo = artigos[slug];
-    new Date(artigo.date).toLocaleDateString('pt-BR');
     listagem.push({ ...artigo, slug});
```

Comment:
Salvando como JSON você deve formatar só na hora de mostrar

```suggestion
    const date = new Date(artigo.date).toLocaleDateString('pt-BR')
    listagem.push({ ...artigo, slug, date });
```